### PR TITLE
Bugfix/829 786

### PR DESCRIPTION
**Pull request by sofiiamatsyhin**

### CODE REVIEW

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss [hunk 1]
```diff
@@ -26,3 +26,10 @@ hr {
     list-style: none;
     padding-left: 0;
 }
+
+@media screen and (max-width: 576px) {
+  p {
+    font-size: 18px;
+    color: #313131;
+  }
+}
```
Comment (mplesha) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss:35]: Fix this 'No new line at EOF'

Comment (sofiiamatsyhin) [reply src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss]: fixed

**Comment by mplesha** [comment]
conflicted PR

**Comment by sofiiamatsyhin** [comment]
Resolved

**Pull request merged**